include/bluetooth/bluetooth.h

```diff
@@ -286,36 +292,55 @@ struct bt_le_adv_param {
 
 	/** Maximum Advertising Interval (N * 0.625) */
 	u16_t interval_max;
+
+	/** Peer address */
+	const bt_addr_le_t *peer_addr;
 };
 
 /** Helper to declare advertising parameters inline
   *
   * @param _options   Advertising Options
   * @param _int_min   Minimum advertising interval
   * @param _int_max   Maximum advertising interval
+  * @param _peer_addr Peer address, used only for directed advertising mode
   */
-#define BT_LE_ADV_PARAM(_options, _int_min, _int_max) \
+#define BT_LE_ADV_PARAM(_options, _int_min, _int_max, _peer_addr) \
```

Comment:
This breaks all existing applications using this macro, so if we can avoid changing it that'd be best.

Reply:
We should probably have a different macro for direct adv.